java/HealthVault Mobile/HVSample/src/main/res/layout/weight.xml

```diff
@@ -4,16 +4,28 @@
 	android:layout_height="match_parent">
 
 	<LinearLayout
```

Comment:
Do we even need a container for the graph? The graph is the same size as the linearlayout container so it doesn't seem to be needed. If it is needed then at least change the linear layout to a frame layout which is more light weight